chainbase/src/main/java/org/tron/core/db2/common/TxCacheDB.java

```diff
@@ -288,6 +295,7 @@ private boolean recovery() {
   }
 
   private boolean recovery(int index, Path file) {
+    checkCrc32c(index, file);
```

Comment:
Has this error happened in tests? Why will it happen? Will some error info printed when this kind of errors be found?

Reply:
To catch accidents like disk write errors, service crash etc.

> Will some error info printed when this kind of errors be found?

Yes,  there will be an exception.